backend/chat_messages/consumer.py

```diff
@@ -1,6 +1,6 @@
 import json
 from django.utils import timezone
-from channels.generic.websocket import AsyncWebsocketConsumer
+from channels.generic.websocket import AsyncWebsocketConsumer  # type: ignore
```

Comment:
What do these mean?

Reply:
it tells mypy that it's ok that the given library does not have type hints (and all things pulled from the library will have type `any` instead of erroring)